13/umbraco-cms/fundamentals/data/defining-content/document-type-localization.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Here you will learn how to apply localization for Document Types in Umbraco
+---
+
+# Localizing Document Types
+
+The Umbraco backoffice is fully localized to match the user's [configured language](../users.md). When defining Document Types, you can apply localization to:
+
+- Document Type names and descriptions.
+- Property names and descriptions.
+- Custom property validation messages.
+- Tab and group names.
+
+Setting up localization for Document Types is a two-step process:
+
+- Create the localizations in [user defined language files](../../../extending/language-files.md).
+- Apply the localizations to the Document Types.
+
+{% hint style="info" %}
+
+Everything in this article also applies to defining [Media Types](../creating-media/README.md).
+
+{% endhint %}
+
+## Creating localizations
+
+User defined language files are created in `/config/lang` and must be named `{language}.user.xml` - e.g. `en-us.user.xml`.
```

Comment:
The folder doesn't exist on a clean install 😄 it needs to be created at the very level you're showing here (`config` folder next to the `umbraco` and `wwwroot` folders). If unclear, this might need to be clarified in https://docs.umbraco.com/umbraco-cms/extending/language-files#user-language-files 🤓

Reply:
will do that 💪 Thank you for confirming 😊